eng/packages/http-client-csharp/eng/scripts/Build-Packages.ps1

```diff
@@ -0,0 +1,77 @@
+#Requires -Version 7.0
```

Comment:
I know some might already exist but should these scripts be under another eng/scripts folder in this package? Or should they be in the root eng/scripts folder?

Reply:
Thanks, moved to eng/scripts/typespec.  This made the script location a constant so the common template no longer needs to be told the emitter package location in the repo.